spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/src/main/resources/application.yml

```diff
@@ -0,0 +1,20 @@
+---
+cloudfoundry:
+  api:
+    endpoint: https://api.${host}.springapps.io
```

Comment:
HOST is currently an environment variable just for convenience in setting the URIs. It is not used directly in the code.